molo/commenting/tests/test_models.py

```diff
@@ -32,3 +39,10 @@ def test_parent(self):
         second_comment.save()
         [child] = first_comment.children.all()
         self.assertEqual(child, second_comment)
+
+    def test_auto_remove(self):
+        comment = self.mk_comment('first comment')
+        comment.save()
+        comment_flag = self.mk_comment_flag(comment)
+        comment_flag.save()
+        self.assertTrue(comment.is_removed)
```

Comment:
@miltontony @smn @saeedmb8 
this test is failing and I don't know why.
as far as I can determine a comment_was_flagged signal should be fired when comment_flag is saved but the receiver in models.py doesn't seem to get called